tasks/sep/ink-002-holocene-system-config-upgrade/OVERVIEW.md

```diff
@@ -0,0 +1,57 @@
+# Holocene Hardfork - SystemConfig Upgrade
```

Comment:
In prior tasks which execute this kind of upgrade, we usually end up deleting this file and relying on readme and validation md files. See https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/028-holocene-system-config-upgrade-and-init-multi-chain. You can move the content of this file into the readme doc.